.gitignore

```diff
@@ -65,3 +65,6 @@ release.properties
 
 # Quarkus CLI
 .quarkus
+
+#Dolphin
```

Comment:
What is Dolphin?

Reply:
I'm using Fedora with KDE. Dolphin is the file manager and when I customise the view of the folder it creates a .directory file.

Reply:
Gotcha, thanks

Reply:
If this is a problem, I can remove the customization, in which case no .directory file will be created.

Reply:
Not a problem :)